ibis/expr/timecontext.py

```diff
@@ -127,6 +127,40 @@ def canonicalize_context(
     return begin, end
 
 
+def construct_multi_index_series(
```

Comment:
It also feels to me what the function is an implementation of the pandas backend and therefore, probably don't belong in ibis/expr/timecontext.py module. Is there a place you can put in the pandas module?

Reply:
I thought about this. This could be a util function in general, also putting this in pandas/execution/timecontext.py seems to cause circular dependency. I could revisit.

Reply:
Why would it have a circular dependency?